Fix and make consistent various Zooz Zen 7x configs

dfederm opened the pull request:
This makes the auto-on/off settings consistent for all the Zooz Zen 7x switches and fixes up a couple (adding units, labeling option values, etc).

dfederm commented:
@zwave-js-bot rename commit

dfederm commented:
@zwave-js-bot help

AlCalzone commented:
`fix` should be lowercase in the commit message

dfederm commented:
> `fix` should be lowercase in the commit message

I'm sure it's not a top pri, but might be nice for the bot to tell the author what they did wrong, or have some snippet in the dev docs to help craft proper PRs. Bit of a learning curve to figure out the correct process/style :)

AlCalzone commented on packages/config/config/devices/0x027a/zen71.json:
```diff
@@ -76,21 +76,36 @@
-			"label": "Auto Turn-Off Timer (Minutes)",
-			"description": "How long the switch remains on",
+			"label": "Auto Turn-Off Timer",
+			"description": "Set the number of minutes to automatically turn off switch once it has been turned on (0 to disable).",
```
I've asked @blhoward2 for a review, but I can tell you already that he'll ask you to remove the `(0 to disable)` part. That's what we have `"options"` for.

dfederm replied:
Sure no problem!

blhoward2 commented on packages/config/config/devices/0x027a/zen71.json:
```diff
@@ -76,21 +76,36 @@
-			"label": "Auto Turn-Off Timer (Minutes)",
-			"description": "How long the switch remains on",
+			"label": "Auto Turn-Off Timer",
+			"description": "Set the number of minutes to automatically turn off switch once it has been turned on.",
```
This description is unnecessary. Please remove them throughout the files if the label makes clear what it does. We already define it is in minutes via the displayed unit. This is a broader problem in the original files, so if you can help us clean that up in these we appreciate it.

dfederm replied:
Sure. I'll just go all out and templatize all these, at least the Zen line.

blhoward2 reviewed (changes requested):
Minor changes, otherwise looks good! We appreciate your effort.

dfederm commented:
@blhoward2 at the risk of being confusing, I renamed the PR and rewrote the description to better match the new scope of the PR. Please review at your convenience.

dfederm commented:
@zwave-js-bot fix lint

AlCalzone commented:
@dfederm FYI, force-pushes make it confusing because it doesn't allow us to see what changed from that last review.

dfederm commented:
> @dfederm FYI, force-pushes make it confusing because it doesn't allow us to see what changed from that last review.

Thanks, will keep that in mind. Professionally I use a code review tool which diffs PR iterations regardless of commits so I'm used to just force-pushing to keep rebasing easy :)

AlCalzone commented:
Oh cool, what tool is that?

dfederm commented:
It's called CodeFlow, but it's unfortunately Microsoft-internal. I assume they haven't made it a public product in favor of improving the Azure DevOps PR experience (which also has this iteration diff feature) and hopefully that'll trickle to GitHub PRs at some point too.

dfederm commented:
@blhoward2 Any other feedback or other PRs we're waiting to sequence this behind?

blhoward2 commented:
I've had some health issues so have been down for a couple of weeks. I'm hoping this weekend.

dfederm commented:
Sorry to hear that! Hope everything is OK. Take your time

blhoward2 reviewed (approved):
Looks pretty good. I made some changes for consistency. Note that this requires #3466 to be merged first as I'm reusing a some new master template entries from that PR. So lint will fail until then.

dfederm commented:
@AlCalzone this looks ready for merge
